science/gerbil/Portfile

```diff
@@ -4,6 +4,7 @@ PortSystem          1.0
 PortGroup           cmake 1.0
 PortGroup           github 1.0
 PortGroup           qt5 1.0
+PortGroup           boost 1.0
 
 github.setup        gerbilvis gerbil 5a7705fe1170f812a6cd0e79a1a853f4d8aec2cf
 version             2020-05-06-5a7705f
```

Comment:
Chris, do you want to rev-bump `gerbil` too?

Reply:
yes, of course thanks.